### PR TITLE
Update for version 1.0.0

### PR DESCRIPTION
**Pull request by cyberiakris**
homepage logic updates. cosmetic changes on ui. missing img assets included.

### CODE REVIEW

**Comment by cyberiakris** [comment]
checked

**Comment by cyberiakris** [comment]
approved

**Review by cyberiakris — COMMENTED** [review]
approved

File: .gitignore
Comment (cyberiakris) [inline .gitignore:1]: checked

**Review by cyberiakris — COMMENTED** [review]
approved

File: .gitignore
Comment (cyberiakris) [reply .gitignore:1]: ok

**Review by krisofixo — APPROVED** [review]
all good

**Pull request merged**